gnucash, gnucash-cli: add page (#5936)
tldr-pages/tldr: pages/common/gnucash-cli.md

# gnucash-cli

> A command-line version of GnuCash.

> > More information: <https://manned.org/gnucash-cli>.

Reviewer: ```suggestion
> More information: <https://gnucash.org>.
```

> - Get online quotes for currencies and stocks in the file:

Reviewer: ```suggestion
- Save online quotes for currencies and stocks to a file:
```
Is this still correct?

Author: No, unfortunately. What the command would do is read the data file (`example.gnucash`) and get a list the foreign currencies and stocks in it. Then it would log online quotes for those currencies to the console.

Reviewer: Now I get what the command does. What about this:
```suggestion
- Get quotes for currencies and stocks specified in a file and print them to stdout:
```

`gnucash-cli --quotes get {{path/to/file.gnucash}}`

> - Generate a report:

Reviewer: What does this command do? Right now, it sounds like it's creating a crash or bug report.

Author: Ahh, sorry about that. It generates financial reports from the specified data file (`example.gnucash`), such as a "Profit & Loss", "Balance Sheet", etc and logs the output to the console.

Reviewer: Ah, okay. I thought it could be any arbitrary name. Then how about
```diff
- - Generate a financial report:
+ - Generate a financial report of a specific type, specified by `--name`:
```

Author: This is one I was a bit confused with how to word, as I felt the name of the argument was terribly ambiguous.
I think this is much better.
